website_crm_quick_answer/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=====================================
+Quick answer for website contact form
+=====================================
+
+This module was written to extend the functionality of the website contact form
+to support sending the interested contact an automatic welcome email and allow
+you to customize it.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Go to *Settings > Technical > Automation > Automated actions > Quick response
+  to website contact form* and edit anything there.
+
+Quicker way to just change the email template (something that most likely you
+will want to do):
+
+* Go to *Settings > Technical > Email > Templates*
+* Edit the template called *Quick response to website contact form*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Have an email address in your main company. It will be used as sender
+  address.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/186/10.0
```

Comment:
@wpichler IMO Change target `11.0`

Reply:
how to create a new runbot target ?

Reply:
only change `:target: https://runbot.odoo-community.org/runbot/186/11.0`